Src/WindowsServer/WindowsServer.Shared/WindowsServer.Shared.projitems

```diff
@@ -12,28 +12,15 @@
     <Compile Include="$(MSBuildThisFileDirectory)AssemblyInfo.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AzureInstanceMetadataTelemetryModule.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AzureWebAppRoleEnvironmentTelemetryInitializer.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)AzureRoleEnvironmentTelemetryInitializer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)BuildInfoConfigComponentVersionTelemetryInitializer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)DeveloperModeWithDebuggerAttachedTelemetryModule.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)DeviceTelemetryInitializer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)DomainNameRoleInstanceTelemetryInitializer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\AzureComputeMetadataHeartbeatPropertyProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\AzureMetadataRequestor.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\AzureRoleEnvironmentContextReader.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\DataContracts\AzureInstanceComputeMetadata.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\DeviceContextReader.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\IAzureMetadataRequestor.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\IAzureRoleEnvironmentContextReader.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\Role.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\RoleEnvironment.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\RoleInstance.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\RuntimeBindingObject.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\ServiceRuntime.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\WindowsServerEventSource.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\AzureServiceRuntimeAssemblyLoader.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\MetricManager.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)UnhandledExceptionTelemetryModule.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)FirstChanceExceptionStatisticsTelemetryModule.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UnobservedExceptionTelemetryModule.cs" />
```

Comment:
Can you tell why UnobservedExceptionTelemetryModule is included for netstandard whereas UnhandledExceptionTelemetryModule,FirstChanceExceptionStatisticsTelemetryModule are only for net45?

Reply:
This is purely because `UnobservedExceptionTelemetryModule` compiles & runs (tests) in both netstandard and net45, whereas `UnhandledExceptionTelemetryModule` and `FirstChanceExceptionStatisticsTelemetryModule` only work in net45 without adding effort.

If possible, I would like to keep this PR to *adding netstandard1.6* to the WindowsServer SDK, without removing any currently existing functionality in net45. 

I hope to address any issues about getting *all* functionality to work on both platforms in a future PR.

Reply:
Ok. Please do mention either in changelog or in the linked GH issue, about which features from WindowsServer package is enabled for .netcore apps. We don't want users to think entire WindowsServer package features works for .net core.

Reply:
Agreed, done.